src/main/java/hudson/plugins/msbuild/MsBuildBuilder.java

```diff
@@ -307,7 +308,7 @@ public Descriptor<Builder> getDescriptor() {
     static String[] tokenizeArgs(String args) {
 
         if (args == null) {
-            return null;
+            return new String[]{};
```

Comment:
I really like this change.  Would you be willing to extend the change a little further by adding the `@NotNull` annotation to the return value of the `tokenizeArgs` method?

That's optional but would help callers know that they can trust the return value will never be null

Reply:
Let me know if you decide that you would rather not make that optional addition.  I'm happy to merge this pull request with or without that addition.